app/helpers/emitter_helper.rb

```diff
@@ -12,4 +12,14 @@ def display_emitter(process)
               scope: "decidim.participatory_processes.emitter")
     }
   end
+
+  def emitter_options
```

Comment:
Here you can return directly a map of expected elements 

```suggestion
    def emitter_options
    Decidim::ParticipatoryProcess.where(organization: current_organization).where.not(emitter_name: nil).map do |pp|
      [pp.emitter_name, pp.id, {'data-image': pp.attached_uploader(:emitter).path}]
    end
  end
```